packages/core/src/tracing/spanBuffer.js

```diff
@@ -101,6 +107,7 @@ exports.init = function init(config, _downstreamConnection) {
   initialDelayBeforeSendingSpans = Math.max(transmissionDelay, minDelayBeforeSendingSpans);
   isFaaS = false;
   transmitImmediate = false;
+  ignoreEndpoints = config.tracing?.ignoreEndpoints;
```

Comment:
```suggestion
  ignoreEndpoints = config.tracing.ignoreEndpoints;
```

This should be always defined. See normalizeConfig.

Reply:
Also: Could we please move this up to the other config.tracing usages.